README.md

```diff
@@ -151,3 +159,32 @@ const resolvers = {
   },
 };
 ```
+
+# Typescript
+
+As the library cannot know the structure of your user object you need to add this to the global scope by writing your own "somename.d.ts" you can adapt the following template with whatever user variables that you want have added to the PassportRequest:
+
+```ts
+declare module "graphql-passport" {
+  interface PassportRequest {
+    user?: {
+      username: string;
+    };
+  }
+}
+```
+
+In your GraphQL functions you can the reference the passport context using:
+
+```ts
+const something = async (
+  parent: unknown,
+  data: SomeData,
+  context: PassportRequest
```

Comment:
I guess I was too quick. This is not correct, the context shouldn't be of type `PassportRequest`. I guess we need an additional type or export the `Context` type from `buildContext` as well.

Can you create a new PR for this? @gforge